spoc/models/file_parameter_models.py

```diff
@@ -0,0 +1,60 @@
+"""This file contains data classes for parameters
+of spoc data structures"""
+from typing import Optional, List
+from pydantic import BaseModel
+
+
+class ContactsParameters(BaseModel):
+    """Parameters for multiway contacts"""
+
+    number_fragments: Optional[int] = None
+    metadata_combi: Optional[List[str]] = None
+    label_sorted: bool = False
+    binary_labels_equal: bool = False
+    symmetry_flipped: bool = False
+
+    def __hash__(self) -> int:
+        # get metadata hash
+        if self.metadata_combi is not None:
+            metadata_hash = hash(tuple(self.metadata_combi))
```

Comment:
So, do we really need `metadata_combi` to be a list? It seems to be immutable anyway. Then we don't need `metadata_hash`.